global.rst

```diff
@@ -0,0 +1,22 @@
+.. roles
```

Comment:
Are these new roles intended to be used throughout the course materials? (They are not included in Sphinx?) Then you could add them to the `rst_prolog` in conf.py.
Nobody wants to manually `.. include::` the same file in each RST chapter.
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-rst_prolog

Note that `rst_prolog` is already set in conf.py and you must not overwrite the previous value. It is a string and can be concatenated. The value of `rst_prolog` can be the include directive or it could directly contain the role definitions.

Reply:
At the moment, the filename `global.rst` does not really match the contents. Well, they are definitions globally available in the course.

Reply:
I will use the  `rst_prolog` and the `rst_epilog`